cove_iati/settings.py

```diff
@@ -9,8 +9,26 @@
 
 env = environ.Env(  # set default values and casting
     DB_NAME=(str, os.path.join(BASE_DIR, 'db.sqlite3')),
+    SENTRY_DSN=(str, ''),
 )
 
+# We use the setting to choose whether to show the section about Sentry in the
+# terms and conditions
+SENTRY_DSN = env('SENTRY_DSN')
+
+if SENTRY_DSN:
+    import sentry_sdk
+    from sentry_sdk.integrations.django import DjangoIntegration
+    from sentry_sdk.integrations.logging import ignore_logger
```

Comment:
The imports can't be at the beginning of the file?

Reply:
I'm not sure, this is how it was in lib-cove-web.

Reply:
It is not a blocker. I have approved the PR but I was wondering if it could go at the beginning. Leave them as they are then. Thanks.